python/test/t_NAIS_std.py

```diff
@@ -72,4 +72,4 @@ def four_branch(x):
 Nais_algo.run()
 NAIS_result = Nais_algo.getResult()
 
-assert_almost_equal(NAIS_result.getProbabilityEstimate(), 0.00215688)
+assert_almost_equal(NAIS_result.getProbabilityEstimate(), 0.00227031)
```

Comment:
you reverted the python test

Reply:
Yes indeed, thank you !
The correction has been made